integrations/openshift.md

```diff
@@ -39,585 +39,77 @@ Here's a preview of the Kubernetes Pods dashboard:
 
 {% include image.md src="images/db_kubernetes_pods.png" width="80" %}
 
+## Use OpenShift To Install The Kubernetes Integration
 
-## Kubernetes Quick Install Using Helm
-1. Ensure that you have installed [helm](https://helm.sh/docs/intro/).
-2. Add the Wavefront helm repo:{% raw %}
-```
-helm repo add wavefront https://wavefronthq.github.io/helm/
-helm repo update
-```
-{% endraw %}
-3. To deploy the Wavefront Collector and Wavefront Proxy:
-
-    Using helm 2:{% raw %}
-    ```
-    helm install wavefront/wavefront --name wavefront --set wavefront.url=https://YOUR_CLUSTER.wavefront.com --set wavefront.token=YOUR_API_TOKEN --set clusterName=<YOUR_CLUSTER_NAME> --namespace wavefront
-    ```
-{% endraw %}
-    Using helm 3:{% raw %}
-    ```
-    kubectl create namespace wavefront
-    helm install wavefront wavefront/wavefront --set wavefront.url=https://YOUR_CLUSTER.wavefront.com --set wavefront.token=YOUR_API_TOKEN --set clusterName=<YOUR_CLUSTER_NAME> --namespace wavefront
-    ```
-{% endraw %}
-
-**Note:** The `clusterName` property refers to the Kubernetes cluster, for example, `dev-cluster`. You must set this property. For vSphere Tanzu, add `--set vspheretanzu.enabled=true` along with helm install command.
```

Comment:
...For vSphere Tanzu, include `--set vspheretanzu.enabled=true` in the `helm install` command.